src/engine/enginebuffer.cpp

```diff
@@ -52,6 +51,9 @@ const double kLinearScalerElipsis = 1.00058; // 2^(0.01/12): changes < 1 cent al
 
 const SINT kSamplesPerFrame = 2; // Engine buffer uses Stereo frames only
 
+// Rate at which the playpos slider is updated
+const int kiPlaypositionUpdateRate = 15; // updates per second
```

Comment:
I think we don't use the type prefixed anymore (except for pointers)

```suggestion
const int kPlaypositionUpdateRate = 15; // updates per second
```